pkg/driverbuilder/buildprocessor.go

```diff
@@ -8,7 +8,7 @@ type BuildArchitecture string
 
 const BuildArchitectureX86_64 BuildArchitecture = "x86_64"
 
-var builderBaseImage = "falcosecurity/driverkit-builder:latest" // This is overwritten when using the Makefile to build
+var BuilderBaseImage = "falcosecurity/driverkit-builder:latest" // This is overwritten when using the Makefile to build
```

Comment:
This can be probably avoided if you set the default value for the flag as suggested above.